### ISSUE TITLE
Support for LeEco Le Max 2

### ISSUE THREAD

**Issue by 0o0will0o0** [offensive-security/kali-nethunter#950]

**Comment by 0o0will0o0** [comment]
Would it be possible to make a version for this device? I can try to help in any and every way possible.

**Comment by kimocoder** [comment]
For porting to new devices and adding support, read the NetHunter wiki [here](https://github.com/offensive-security/kali-nethunter/wiki)

The 3 steps needed for complete support would be

* Getting started, setting up a build environment and preparation, read [here](https://github.com/offensive-security/kali-nethunter/wiki/Porting-Nethunter)

* Modifying & building a kernel, howto? Read [here](https://github.com/offensive-security/kali-nethunter/wiki/Modifying-the-Kernel)

* Adding device support, read [here](https://github.com/offensive-security/kali-nethunter/blob/master/nethunter-installer/README.md)

**Comment by 0o0will0o0** [comment]
If find those pages to be incredibly vague, give little to no instruction on how to make a modified kernel for any device. IMO, the porting nethunter instructions are useless for help or guide.

**Comment by kimocoder** [comment]
Yes the wiki should be updated on several places.
The right people to announce on this matter (both porting & wiki) would be @jcadduono or @binkybear

**Comment by 0o0will0o0** [comment]
@jcadduono I would very much like some help with porting nethunter on my phone's kernel. I am now using AOSP Extended ROM with msm8996 cm 14.1 kernel. I tried following your instructions on the wiki but it's beyond vague for a newbie like me. So far I've downloaded the toolchain through Kali via the terminal and changed the marshmallow bit to nougat, but when I went to the "Modifying the Kernel" section, I had no idea where to find my defconfig, where to put my working folder, how to work with the Kernel Configuration tool, and how to pack it with the chroot
